packages/material-ui-codemod/src/v1.0.0/import-path.test/expected.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { withStyles } from 'material-ui/styles'
+import MenuItem from 'material-ui/MenuItem';
+import Tab from 'material-ui/Tab';
+import MuiTabs from 'material-ui/Tabs';
+import BottomNavigationAction from 'material-ui/BottomNavigationAction';
+import BottomNavigation from 'material-ui/BottomNavigation';
+import CardContent from 'material-ui/CardContent';
+import CardActions from 'material-ui/CardActions';
+import Card from 'material-ui/Card';
+import CardMedia from 'material-ui/CardMedia';
+import CardHeader from 'material-ui/CardHeader';
+import MuiCollapse from 'material-ui/Collapse';
+import ListItemSecondaryAction from 'material-ui/ListItemSecondaryAction';
+import ListItemText from 'material-ui/ListItemText';
+import ListItemAvatar from 'material-ui/ListItemAvatar';
+import ListItem from 'material-ui/ListItem';
+import ListItemIcon from 'material-ui/ListItemIcon';
+import List from 'material-ui/List';
+import DialogTitle from 'material-ui/DialogTitle';
+import Dialog from 'material-ui/Dialog';
+import DialogContentText from 'material-ui/DialogContentText';
+import DialogContent from 'material-ui/DialogContent';
+import DialogActions from 'material-ui/DialogActions';
+import withMobileDialog from 'material-ui/withMobileDialog';
+import Slide from 'material-ui/Slide';
+import RadioGroup from 'material-ui/RadioGroup';
+import Radio from 'material-ui/Radio';
+import FormControlLabel from 'material-ui/FormControlLabel';
+import ExpansionPanelActions from 'material-ui/ExpansionPanelActions';
+import ExpansionPanelDetails from 'material-ui/ExpansionPanelDetails';
+import ExpansionPanelSummary from 'material-ui/ExpansionPanelSummary';
+import ExpansionPanel from 'material-ui/ExpansionPanel';
+import GridListTile from 'material-ui/GridListTile';
+import GridList from 'material-ui/GridList';
+import CircularProgress from 'material-ui/CircularProgress';
+import MuiLinearProgress from 'material-ui/LinearProgress';
```

Comment:
Small glitch?

Reply:
It's a stress test. Why?

Reply:
None of the other imports are prefixed, so...

Reply:
People might be aliasing the modules:
`import foo from 'bar'` or `import { foo as bar} from 'baz'`.

Reply:
I still don't get why `MuiLinearProgress` rather than just `LinearProgress`, but whatever.

Reply:
It tests that the following transformation is done:
**input**
```js
import { LinearProgress as MuiLinearProgress } from 'material-ui/Progress';
```
**output**
```
import MuiLinearProgress from 'material-ui/LinearProgress';
```